portfolio/src/main/webapp/script.js

```diff
@@ -69,3 +71,27 @@ function scrollToElement(elementId, duration = 300)
                     scrollTop: $('#' + elementId).offset().top - navOffset
                 }, duration);
 }
+
+/**
+ * Function that loads comments
+ */
+function LoadComments() {
+    fetch('/data').then(response => response.json()).then((comments) => {
+        const commentsElement = document.getElementById('comments');
+        commentsElement.innerHTML = '';
+        for(let comment of comments) {
+            commentsElement.appendChild(
+                createListElement('Message: ' + comment.message +
+                                  ', posted on: ' + comment.addedDate));
+        }
+    });
+}
+
+/** 
+ * Function that creates an <li> element containing text
+ */
+function createListElement(text) {
+  const liElement = document.createElement('li');
+  liElement.innerText = text;
```

Comment:
I'm a bit confused by the 'const' keyword. What does it mean in this context? What purpose does it serve?

Reply:
'const' is like declaring a variable in the local scope, but, unlike 'let', you are not allowed to change its value later. So I will get an error if I try to change liElement's value.